src/ducks/Studio/timeseries/hooks.js

```diff
@@ -124,12 +127,14 @@ export function useTimeseries (
           return [...loadingsSet]
         })
 
+        const strictInterval = hasDaaMetric ? '1d' : interval
+
         client
           .query({
             query,
             variables: {
               metric: key,
-              interval,
+              interval: strictInterval,
```

Comment:
`useTimeseries` is and should be only responsible for fetching and aggregating data. It has a public interface for tweaking metric-specific query arguments – `MetricSettingMap` (third argument)

Reply:
@DmitriVanGuard , :(
A lot of places with usage of useTimeseries. Programmer can forgor about DAA interval if he will add any chart with this hook.
![image](https://user-images.githubusercontent.com/14061779/81173184-50bc0b00-8fa8-11ea-9c6b-f388bb03cb7b.png)

Reply:
Changed(CUSTOM_METRIC_SETTINGS_MAP), need check